doc/spire_server.md

```diff
@@ -59,8 +59,8 @@ This may be useful for templating configuration files, for example across differ
 | `ca_subject`            | The Subject that CA certificates should use (see below)                                                                                                                                                                                             |                                                                |
 | `ca_ttl`                | The default CA/signing key TTL                                                                                                                                                                                                                      | 24h                                                            |
 | `data_dir`              | A directory the server can use for its runtime                                                                                                                                                                                                      |                                                                |
-| `default_x509_svid_ttl` | The default X509-SVID TTL
-| `default_jwt_svid_ttl`  | The default JWT-SVID TTL
+| `default_x509_svid_ttl` | The default X509-SVID TTL | 1h |
+| `default_jwt_svid_ttl`  | The default JWT-SVID TTL | 5m |
```

Comment:
can you align this table?

Reply:
I see @amartinezfayo aligned these locally to themselves, but they need to be aligned with all the other defaults.